ajax/libs/startbootstrap-sb-admin-2/package.json

```diff
@@ -2,26 +2,27 @@
   "name": "startbootstrap-sb-admin-2",
   "filename": "js/sb-admin-2.min.js",
   "version": "3.3.7",
-  "homepage": "http://startbootstrap.com/template-overviews/sb-admin-2/",
+  "homepage": "https://startbootstrap.com/template-overviews/sb-admin-2/",
   "description": "A free, open source, Bootstrap admin theme created by Start Bootstrap",
   "keywords": [
     "bootstrap",
     "theme"
   ],
-  "license": "Apache",
+  "license": "MIT",
   "autoupdate": {
     "source": "git",
-    "target": "git://github.com/IronSummitMedia/startbootstrap-sb-admin-2.git",
+    "target": "git://github.com/BlackrockDigital/startbootstrap-sb-admin-2.git",
     "basePath": "dist/",
     "files": [
       "js/sb-admin-2.js",
+      "js/sb-admin-2.min.js",
       "css/sb-admin-2.css",
-      "css/timeline.css"
+      "css/sb-admin-2.min.css"
```

Comment:
I'm not really planning on any more structural changes at this point, but I could do css/**/* and js/**/* - that does make sense too.